Procfile.tmpl

```diff
@@ -1,4 +1,5 @@
 thin:            bundle exec rails s -p $PORT
 sidekiq:         bundle exec sidekiq -q miq_bot
+sidekiq:         bundle exec sidekiq -q miq_bot_glacial
```

Comment:
It doesn't *have* to be, but it makes reading the log much harder if it's not.  @bdunne Please change.